From 21d67338f24455be6672f2d636b44b030d20e170 Mon Sep 17 00:00:00 2001
From: Ayyappa Ch <ayyappa.chandolu@amd.com>
Date: Fri, 27 Jan 2017 15:58:23 +0530
Subject: [PATCH 1663/1722] drm/amdgpu: Use AMDGPU_FENCE_OWNER_VM in unmap

Temporary commit until a permanent solution is provided by the amdgpu
team. For KFD applications VM updates should ignore KFD eviction fence.
If this change is not there then there is no easy way to identify VM map
and unmaps.

Change-Id: I23212f42803f2232e35ad5d5baceced0ab24432a
Signed-off-by: Harish Kasiviswanathan <Harish.Kasiviswanathan@amd.com>
Signed-off-by: Ayyappa Ch <ayyappa.chandolu@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
index 1e40074..7cceb22 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
@@ -1041,10 +1041,6 @@ static int amdgpu_vm_bo_update_mapping(struct amdgpu_device *adev,
 	params.adev = adev;
 	params.src = src;
 
-	/* sync to everything on unmapping */
-	if (!(flags & AMDGPU_PTE_VALID))
-		owner = AMDGPU_FENCE_OWNER_UNDEFINED;
-
 	/* If update flag is set to CPU, then try to update PT entries directly
 	 * by CPU. If failed, fallback to SDMA update.
 	 */
-- 
2.7.4

